pulumi/infra/docker_images.py

```diff
@@ -16,6 +16,18 @@
 - an image pulled from Cloudsmith
     "docker.cloudsmith.io/grapl/raw/graph-merger:20211105192234-a86a8ad2"
 """
+DockerImageVersion = NewType("DockerImageId", str)
```

Comment:
Should this be `NewType("DockerImageVersion", str)` instead?

Reply:
Yes 🤦